app/src/main/java/com/kickstarter/libs/utils/AnimationUtils.kt

```diff
@@ -20,4 +26,52 @@ object AnimationUtils {
     animation.fillAfter = true
     return animation
   }
+
+  @JvmOverloads
+  fun fadeInAndScale(view: View, duration: Long = 500L, startDelay: Long = 500L,
```

Comment:
Open to renaming suggestions.

Reply:
Do we see this being used elsewhere? If so then the name is fine but if it's just going to be used for the heartbeat animation I would name it something related to it.